src/server/hiring_process.go

```diff
@@ -208,3 +220,76 @@ func VacancyTable(
 		c.JSON(http.StatusOK, vacancies)
 	}
 }
+
+// ListDepartments godoc
+// @Summary List departments
+// @Schemes
+// @Description Return a list of departments with id and title
+// @Tags departments
+// @Produce json
+// @Success 200 {array} model.Suggestion
+// @Router /departments [get]
+func ListDepartments(
+	client *ent.Client,
+) func(c *gin.Context) {
+	return func(c *gin.Context) {
+		departments, err := service.ListDepartments(c, client)
+		if err != nil {
+			c.JSON(http.StatusInternalServerError, gin.H{
+				"error":   "Failed to list departments",
+				"details": err.Error(),
+			})
+			return
+		}
+		c.JSON(http.StatusOK, departments)
+	}
+}
+
+// CreateGroupAcess godoc
+// @Summary Create a new group access
+// @Schemes
+// @Description Create a new group access with name and related departments
+// @Tags group_acess
+// @Accept json
+// @Produce json
+// @Param body body service.CreateGroupAcessRequest true "Group Access Info"
+// @Success 201 {object} ent.GroupAcess
+// @Router /group_acess [post]
```

Comment:
essa rota do swagger não condiz com a rota que está sendo criada, o certo seria:

```suggestion
// @Router /groupAccess/groupAccess [post]
```

ou melhor ainda, deixando padronizado (da forma que estão todos os outros endpoints) na questão de camel case vs pascal case:

```suggestion
// @Router /group_access/group_access [post]
```

melhor ainda, reduzindo repetição:

```suggestion
// @Router /group_access [post]
```
<br>

> [!NOTE]
segunda e terceira sugestão envolvem trocar na linha 42 tbm

> [!NOTE]
no caso da terceira sugestão, na linha 42 ficaria `groupAccess.POST("", CreateGroupAcess(dbClient))` (string vazia como `path`)